src/en/news/blog/2023/v18-2-0-reef-released/index.md

```diff
@@ -0,0 +1,878 @@
+  description’`` (entering your own cluster description).
+
+## <a id="upgrade"></a>Upgrading from Pacific or Quincy
+
+Before starting, make sure your cluster is stable and healthy (no down or recovering OSDs). (This is optional, but recommended.) You can disable the autoscaler for all pools during the upgrade using the noautoscale flag.
+
+**Note**: You can monitor the progress of your upgrade at each stage with the `ceph versions` command, which will tell you what ceph version(s) are running for each type of daemon.
+
+### Upgrading cephadm clusters
+
+If your cluster is deployed with cephadm (first introduced in Octopus), then the upgrade process is entirely automated. To initiate the upgrade,
+
+    ceph orch upgrade start --image quay.io/ceph/ceph:v18.2.0
+
+The same process is used to upgrade to future minor releases.
+
+Upgrade progress can be monitored with
+
+    ceph orch upgrade status
+
+Upgrade progress can also be monitored with `ceph -s` (which provides a simple progress bar) or more verbosely with
+
+    ceph -W cephadm
+
+The upgrade can be paused or resumed with
+
+    ceph orch upgrade pause  # to pause
+    ceph orch upgrade resume # to resume
+
+or canceled with
+
+    ceph orch upgrade stop
+
+Note that canceling the upgrade simply stops the process; there is no ability to downgrade back to Pacific or Quincy.
+
+### Upgrading non-cephadm clusters
+
+**Note**: If you cluster is running Octopus (15.2.x) or later, you might choose to first convert it to use cephadm so that the upgrade to Quincy is automated (see above). For more information, see https://docs.ceph.com/en/reef/cephadm/adoption/.
+
+1. Set the `noout` flag for the duration of the upgrade. (Optional, but recommended.)
+
+   ceph osd set noout
+
+2. Upgrade monitors by installing the new packages and restarting the monitor daemons. For example, on each monitor host,
+
+   systemctl restart ceph-mon.target
+
+   Once all monitors are up, verify that the monitor upgrade is complete by looking for the `reef` string in the mon map. The command
+
+   ceph mon dump | grep min_mon_release
+
+   should report:
+
+   min_mon_release 18 (reef)
+
+   If it doesn't, that implies that one or more monitors hasn't been upgraded and restarted and/or the quorum does not include all monitors.
+
+3. Upgrade `ceph-mgr` daemons by installing the new packages and restarting all manager daemons. For example, on each manager host,
+
+   systemctl restart ceph-mgr.target
+
+   Verify the `ceph-mgr` daemons are running by checking `ceph -s`:
+
+   ceph -s
+
+   ...
+   services:
+   mon: 3 daemons, quorum foo,bar,baz
+   mgr: foo(active), standbys: bar, baz
+   ...
+
+4. Upgrade all OSDs by installing the new packages and restarting the ceph-osd daemons on all OSD hosts
+
+   systemctl restart ceph-osd.target
+
+5. Upgrade all CephFS MDS daemons. For each CephFS file system,
+
+   1. Disable standby_replay:
+
+      ceph fs set <fs_name> allow_standby_replay false
+
```

Comment:
@batrick don't we need to set `mon_mds_skip_sanity = true` here?

Reply:
That should only be necessary if upgrading from a Pacific `<=16.2.5`. But since that's a valid version to upgrade from, we should make note of it.

I also see now that we're always doing that config change for every upgrade. I don't think that should be the case but I don't feel comfortable backporting that change to reef or earlier. I'll make a ticket for Squid.

Reply:
Will make note of this

Reply:
@dvanders I added this as step 2; please verify

Reply:
Verified